comparecomplyv1/compare_comply_v1_integration_test.go

```diff
@@ -3,7 +3,7 @@
 package comparecomplyv1_test
 
 /**
- * (C) Copyright IBM Corp. 2018, 2019.
+ * (C) Copyright IBM Corp. 2018, 2020.
```

Comment:
same comment about lint copyright dates - i wont comment on these from this point